agent/util-scripts/gold/test-client-tool-meister/test-53.txt

```diff
@@ -1,8 +1,6 @@
 +++ Running test-53 test-client-tool-meister 
 "mpstat" tool is now registered for host "testhost.example.com" in group "default"
 "dcgm" tool is now registered for host "testhost.example.com" in group "default"
-pbench-tool-data-sink: connected to redis server localhost:17001
-pbench-tool-meister: connected to redis server localhost:17001
```

Comment:
Why do you have this change in your PR?

Reply:
It's not there now. Transient rebase artifact?? Weird.